script/Arcueid/Day 12/12_00_ARC12_2.txt

```diff
@@ -791,7 +791,7 @@ I remember the words scratched into the gazebo.
 [sha:4f95614cf71ce2f7091940eccddb08fa0ba2e56c]{
 -- Page 75, Offset 30748.
 -- この肉体に転生した吸血鬼。姫君の敵対者。
-The vampire that resurrected into this body. Adversary of the Princess.
+The vampire who resurrected into this body――the Princess's adversary.
```

Comment:
Hmm, I do think "Adversary of the Princess" is more of a 'title', and thus makes more sense with the following line.

Reply:
Well, "the Princess's adversary" can serve much the same purpose and it feels like it links better with the prior line

Reply:
I guess